libcodechecker/server/client_db_access_handler.py

```diff
@@ -124,18 +250,31 @@ def __lastBugEventPos(self, report_id):
         belongs to the given report. If no such event is found then None
         returns.
         """
-        last = self.__session.query(BugPathEvent) \
-            .filter(BugPathEvent.report_id == report_id) \
-            .order_by(BugPathEvent.order.desc()) \
-            .limit(1).one_or_none()
+        try:
+            session = self.__Session()
+
+            last = session.query(BugPathEvent) \
+                .filter(BugPathEvent.report_id == report_id) \
+                .order_by(BugPathEvent.order.desc()) \
+                .limit(1).all()
 
-        if not last:
-            return None
+            if len(last) < 1:
```

Comment:
If it is less than 1? Considering `last` is a database result whose length is a natural number, saying `if len(last) == 0` would look better.